python/lsst/meas/deblender/baseline.cc

```diff
@@ -0,0 +1,116 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ * See the COPYRIGHT file
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "lsst/afw/image/Image.h"
+#include "lsst/afw/image/MaskedImage.h"
+#include "lsst/afw/detection/Footprint.h"
+#include "lsst/afw/detection/Peak.h"
+
+#include "lsst/meas/deblender/Baseline.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace deblender {
+
+namespace{
+    template <typename ImagePixelT,
+              typename MaskPixelT=lsst::afw::image::MaskPixel,
+              typename VariancePixelT=lsst::afw::image::VariancePixel>
```

Comment:
Note that in previous cases we have just used the defaults and only kept `PixelT`.